routes/user.js

```diff
@@ -284,6 +284,7 @@ router.post("/acceptFriendRequest", requireUserAuth, async (req, res) => {
       { _id: friendId },
       { $push: { friends: userId } }
     );
+    friend.outgoingFriendRequests.pull = userId;
```

Comment:
We can remove this line, think it might be an artifact from rebase

Reply:
Cool I took that out and as for combining the expiredListings.helper to cronJobs.helper we could probably do that during our refactoring sprint

Reply:
Good idea, everything else looks good to me!